src/app/store/mod.rs

```diff
@@ -128,7 +128,7 @@ impl Deref for StateWrapper {
 
 impl StateWrapper {
     pub fn msg(&mut self, msg: &Msg) {
-        &self.0.msg(msg);
+        let _ = &self.0.msg(msg);
```

Comment:
This can just be `self.0.msg(msg);`

Reply:
@chinedufn Hi, this generates cargo warning